### PR TITLE
Updated contributing guidelines

### PR DESCRIPTION
**Pull request by nijotz**
## CHANGELOG
* Updated Contributing Guidelines file

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -1,15 +1,60 @@
...
+### Feature Requests
+
+Feature requests should be submitted in the [Issue tracker](../../issues), with
+a description of the expected behavior & use case. Before submitting an Issue,
+please search for similar ones in the [closed
+issues](../../issues?q=is%3Aissue+is%3Aclosed+label%3Aenhancement).
+
+### Pull Requests
+
+We can only accept PRs for version v3.0.1 or greater due to open source
```
Comment (michaelgoin) [inline CONTRIBUTING.md]: Believe this will be `v3.1.0` if we stick with doing license change as a minor version.

File: CONTRIBUTING.md [hunk 2]
```diff
@@ -1,15 +1,60 @@
...
+\#oss-contributors channel in the newrelicusers.slack.com workspace. To request
+access, see https://newrelicusers-signup.herokuapp.com/.
+
+## The Code
+
+### Version Support
+
 When contributing, keep in mind that New Relic customers (that's you!) are
 running many different versions of Node, some of them pretty old. Changes that
 depend on the newest version of Node will probably be rejected, with prejudice
```
Comment (michaelgoin) [inline CONTRIBUTING.md]: "rejected, with prejudice" lol. lets use this as an opportunity to have more user friendly language. it is also less of a concern now that we only keep 3 versions of Node. perhaps we axe this whole section. at a minimum, could have nicer language such as "changes that break currently supported versions of Node are likely to result in requested updates for backwards compatibility."

**Review by michaelgoin — COMMENTED** [review]
Looks pretty good.

Think we need to modify the expected version.

Beyond that, a recommendation to fix some old language that could be a little more welcoming. :)

**Comment by michaelgoin** [comment]
I've also changed this PR to target the interim branch.

File: CONTRIBUTING.md (hunk 2 above)
Comment (nijotz) [reply CONTRIBUTING.md]: I'll take out the whole section

File: CONTRIBUTING.md (hunk 1 above)
Comment (nijotz) [reply CONTRIBUTING.md]: Updated

**Review by michaelgoin — APPROVED** [review]
👍

**Pull request merged**